### PR TITLE
feat(dashboard): connect api with name-edit

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

SSIA

### Things to Talk About

Sorry for late song-lang. I'll add translation by myself later.

### CODE REVIEW

File: src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue [hunk 1]
```diff
@@ -89,28 +92,38 @@ export default defineComponent<Props>({
...
+        const loadDashboard = async () => {
+            if (isProjectDashboard) {
+                await store.dispatch('dashboard/loadProjectDashboard');
+            } else {
+                await store.dispatch('dashboard/loadDomainDashboard');
             }
         };
 
         const handleConfirm = () => {
             updateDashboard();
```
Comment (yuda110) [inline src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue:125]: suggestion: I think that this needs to be async.

```suggestion
        const handleConfirm = async () => {
            await updateDashboard();
```

**Review by WANZARGEN — APPROVED** [review]
The small PR unit is good to review :)
I left a comment!

File: src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue [hunk 2]
```diff
@@ -89,28 +92,38 @@ export default defineComponent<Props>({
...
+                    });
                 } else {
-                    // await SpaceConnector.client.costAnalysis.publicDashboard.update({
-                    //     public_dashboard_id: props.dashboardId,
-                    //     name: name.value,
-                    // });
+                    await SpaceConnector.clientV2.dashboard.domainDashboard.update({
+                        domain_dashboard_id: props.dashboardId,
+                        name: name.value,
+                    });
```
[the remark this answers is not in the document]
Comment (minseolee) [reply src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue:108]: I've asked about it.
And It has been decided to showing only fail message!

File: src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue (hunk 1 above)
Comment (minseolee) [reply src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue:125]: Oh, I forgot it. Thanks a lot

**Pull request merged**